recipe/meta.yaml

```diff
@@ -62,34 +76,34 @@ outputs:
 
   - name: libmambapy
     version: {{ libmambapy_version }}
-    script: build_mamba.sh                       # [unix]
-    script: build_mamba.bat                      # [win]
+    script: build_libmambapy.sh                   # [unix]
+    script: bld_libmamba.bat                      # [win]
     build:
       string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
```

Comment:
We should skip `py<37` for `libmambapy`, https://github.com/mamba-org/mamba/blob/4b663ada019715ad4c989a1a57e81fe12edde3d8/libmambapy/setup.cfg#L17

Reply:
Added.